CGATPipelines/pipeline_enrichment.py

```diff
@@ -1,8 +1,143 @@
-"""===========================
-Pipeline Enrichment
-===========================
-
-
+'''
+PipelineEnrichmentGSEA.py
+=============================================
+
+:Tags: Python
+
+Usage
+-----
```

Comment:
The same way that you have a separate section in the documentation below:
```
----------------------------------------------------------------------------
       Enrichment analysis (Overrepresentative analysis(ORA))
----------------------------------------------------------------------------
```

There should be a similar section here for the GSEA section:
```
----------------------------------------------------------------------------
       Gene Set Enrichment Analysis: GSEA
----------------------------------------------------------------------------
```
Moreover, I suggest you update the documentation further so it is clear that this is the documentation for pipeline script rather than the module file.